recipes-browser/wpewebkit/wpewebkit_2.31.91.bb

```diff
@@ -1,20 +0,0 @@
-require wpewebkit.inc
-require conf/include/devupstream.inc
-
-FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
-SRC_URI = "\
-    https://wpewebkit.org/releases/${BPN}-${PV}.tar.xz \
-    file://0001-EME-Fix-KeySystem-permission-handling.patch \
-    file://274943-GStreamer-Use-imxvideoconvert_g2d-element-inside-the.patch \
```

Comment:
I am talking about these.

Reply:
0001-EME-Fix-KeySystem-permission-handling.patch removed.
274943-GStreamer-Use-imxvideoconvert_g2d-element-inside-the.patch still in use by   wpewebkit_2.30.6.bb.